docs/ops/detection/Proposal_4.md

```diff
@@ -0,0 +1,160 @@
+  * **Range of values**: a positive integer number
+  * **Type**: `int`
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *ratio*
+
+  * **Description**: *ratio* is the ratios for anchor generation.
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *scale*
+
+  * **Description**: *scale* is the scales for anchor generation.
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *clip_before_nms*
+
+  * **Description**: *clip_before_nms* flag that specifies whether to perform clip bounding boxes before non-maximum suppression or not.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: True
+  * **Required**: *no*
+
+* *clip_after_nms*
+
+  * **Description**: *clip_after_nms* is a flag that specifies whether to perform clip bounding boxes after non-maximum suppression or not.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+
+* *normalize*
+
+  * **Description**: *normalize* is a flag that specifies whether to perform normalization of output boxes to *[0,1]* interval or not.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+
+* *box_size_scale*
+
+  * **Description**: *box_size_scale* specifies the scale factor applied to logits of box sizes before decoding.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: 1.0
+  * **Required**: *no*
+
+* *box_coordinate_scale*
+
+  * **Description**: *box_coordinate_scale* specifies the scale factor applied to logits of box coordinates before decoding.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: 1.0
+  * **Required**: *no*
+
+* *framework*
+
+  * **Description**: *framework* specifies how the box coordinates are calculated.
+  * **Range of values**:
+    * "" (empty string) - calculate box coordinates like in Caffe*
+    * *tensorflow* - calculate box coordinates like in the TensorFlow* Object Detection API models
+  * **Type**: string
+  * **Default value**: "" (empty string)
+  * **Required**: *no*
+
+**Inputs**:
+
+*   **1**: 4D input floating point tensor with class prediction scores. Required.
+
+*   **2**: 4D input floating point tensor with box logits. Required.
+
+*   **3**: 1D input floating tensor 3 or 4 elements:  [`image_height`, `image_width`, `scale_height_and_width`] or [`image_height`, `image_width`, `scale_height`, `scale_width`]. Required.
+
+**Outputs**:
+
+*   **1**: Floating point tensor of shape `[batch_size * post_nms_topn, 5]`.
+
+*   **2**: 1D floating point tensor of size `post_nms_topn` with probabilities. *Optional*
+
+**Example**
```

Comment:
Please, add a section called "Types". Refer to https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sort/NonMaxSuppression_4.md for an example.

Reply:
Done.